src/test/java/seedu/address/testutil/TypicalExpenses.java

```diff
@@ -61,6 +63,7 @@ public static ExpenseExpert getTypicalExpenseExpert() {
         for (Expense expense : getTypicalExpenses()) {
             ee.addExpense(expense);
         }
+        ee.setBudget(MAY_BUDGET);
```

Comment:
Should this be here as the class seem to be producing expenses only

Reply:
The class returns a ExpenseExpense, which in my opinion, should include budget to pass less things around the functions.